.gitignore

```
@@ -3,4 +3,8 @@
 .packages
 build/
 # If you're building an application, you may want to check-in your pubspec.lock
-pubspec.lock
\ No newline at end of file
+pubspec.lock
+
+# vladchuk
+.idea
+resources/save.properties
```

What is "save.properties". If you need test data, it can be added to the test code and shouldn't be gitignored. That way, anyone working on the repo can run the same tests. 
```suggestion
.idea
```